geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/collections/SizeableObject2ObjectOpenCustomHashMapWithCursor.java

```diff
@@ -29,52 +33,58 @@
  * The scan method provides the same guarantees as Redis's HSCAN, and in fact
  * uses the same algorithm.
  */
-public class Object2ObjectOpenCustomHashMapWithCursor<K, V>
-    extends Object2ObjectOpenCustomHashMap<K, V> {
+public class SizeableObject2ObjectOpenCustomHashMapWithCursor<K, V>
+    extends Object2ObjectOpenCustomHashMap<K, V> implements Sizeable {
+
+  private static final long serialVersionUID = 9079713776660851891L;
+  public static final int BACKING_ARRAY_OVERHEAD_CONSTANT = 136;
+  public static final int BACKING_ARRAY_LENGTH_COEFFICIENT = 4;
+  private static final ReflectionSingleObjectSizer elementSizer =
+      ReflectionSingleObjectSizer.getInstance();
 
+  private int keysOverhead;
+  private int valuesOverhead;
```

Comment:
I'm okay with having two overhead variables but I don't see any need for two. It looks like you could just have one "sizeInBytes" that tracks both keys and values. If you initialized it to BACKING_ARRAY_OVERHEAD_CONSTANT then the getSizeInBytes could just be "return sizeInBytes;"

Reply:
Good call, I've combined the two.